### PR TITLE
Wordlists suggestions

### PR DESCRIPTION
**Pull request by fogti**
###### Motivation for this change

complexity and delay in https://github.com/NixOS/nixpkgs/pull/104712

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fogti** [comment]
ok, I'm now going to test these changes.

**Comment by fogti** [comment]
I tested `nix-shell -p "(import ./. {}).security-wordlists.withLists (ls: [ls.wfuzz])"`. works.

**Comment by fogti** [comment]
I'm currently testing `nix-shell -p "(import ./. {}).security-wordlists.withLists (ls: with ls; [dirbuster nmap rockyou seclists wfuzz])"`
Seems to work so far...

File: pkgs/tools/security/wordlists/rockyou.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    mkdir src
+    gzip -dc ${src} > src/rockyou.txt
+  '';
+
+  installPhase = ''
+    install -m 444 -D rockyou.txt $out/share/rockyou.txt
+  '';
+
+  meta = with lib; {
+    license = "unknown";
```
Comment (jonringer) [inline pkgs/tools/security/wordlists/rockyou.nix]: would be nice if the license was known. :(

**Comment by fogti** [comment]
@Pamplemousse could you please merge this, or are there any remaining concerns?

**Pull request closed without merging**